static/js/shared/util.ts

```diff
@@ -14,6 +14,17 @@
  * limitations under the License.
  */
 
+export const topics = [
```

Comment:
add a note that this needs to stay in sync with server/__init__.py?

Reply:
Done